lib/stripe_mock/request_handlers/tokens.rb

```diff
@@ -16,15 +16,13 @@ def create_token(route, method_url, params, headers)
 
         cus_id = params[:customer]
 
-        if cus_id && params[:source]
-          customer = assert_existence :customer, cus_id, customers[stripe_account][cus_id]
+        customer = assert_existence :customer, cus_id, customers.dig(stripe_account, cus_id) || customers[Stripe.api_key][cus_id]
```

Comment:
When [creating a token for cloning sources](https://stripe.com/docs/connect/cloning-customers-across-accounts), it's needed to create a token with a platform customer and a stripe account, so it isn't always true that the customer has to live in the same account.